05-promoting_diversity.Rmd

```diff
@@ -148,30 +149,33 @@ Recent projects in which participants are recruited through community engagement
 1) Provide adequate financial, logistical, and time support
 
 Researchers should consider how participants can cope with healthcare needs following participation in a study.
-Investigators should consider how to support caregivers who have important obligations to tend to their family members or others, for example when possible, perhaps home visits would be feasible. Other strategies include providing flexible hours for trial visits and support for transportation, such as payment for a rideshare service or taxi.[@ACRP_representation_2020] Researchers should also consider providing patients with a cell phone to improve communication.[@clark_increasing_2019]
+Investigators should consider how to support caregivers who have important obligations to tend to their family members or others, for example when possible, perhaps home visits would be feasible.
```

Comment:
I'm a bit confused on the flow of this paragraph and how this sentence is/is not related to the one before it. Feels a little like its a separate point?